### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by maclover7**
#219

### CODE REVIEW

**Comment by tute** [comment]
That's perfect, thank you @maclover7! Do you think there's a way to keep a `4.1-protected-attributes` case, or `4.2-protected-attributes`?

**Comment by tute** [comment]
Regarding the failure, I believe some versions of Rails don't play well with older Rubies. Check the `allowed_failures` in doorkeeper's travis.yml.

**Comment by maclover7** [comment]
^ Yep, just realized that Rails 3.2 fails on Ruby 2.2

**Comment by maclover7** [comment]
Updated, waiting on Travis to hear back.

What do you need protected attributes for?

**Comment by tute** [comment]
Some people upgrade Rails 3.2 into 4, but continue using the `attr_accessible` syntax, for a smoother, step by step upgrade. There's specific logic in merit to take that into account, and those conditionals should be tested.

**Comment by maclover7** [comment]
In what version is ```attr_accessible``` deprecated?

**Comment by tute** [comment]
In Rails 4: http://www.easyactiverecord.com/blog/2014/04/01/rails4-strong-parameters-and-the-attr-accessible-macro/. Those people add the `protected_attributes` gem so the Models still understand the call. And there's weird corner cases around that.

File: .travis.yml [hunk 1]
```diff
@@ -2,20 +2,17 @@ language: ruby
...
   - RAILS_VERSION=3.2
-  - RAILS_VERSION=4.0
-  - RAILS_VERSION=4.0-protected-attributes
   - RAILS_VERSION=4.1
   - RAILS_VERSION=4.2
   - ORM=active_record
 allow_failures:
   - ORM=mongoid
+  - env: rails=3.2.0
+    rvm: 2.2
```
Comment (tute) [inline .travis.yml]: Knowing that we expect it to always fail, I'd put it in an `exclude` block. We fix global warming one step at a time! :)

**Comment by tute** [comment]
Merged in https://github.com/merit-gem/merit/pull/221. Thank you @maclover7!

**Pull request closed without merging**